src/writers/file_writer/header_writer/mod.rs

```diff
@@ -28,15 +26,25 @@ impl HeaderBuilder {
     fn get_import_string() -> String {
         // TODO: this method might be better if it read data in from another source
         let parent_actix_import = "actix_web";
-        let actix_imports = ["middleware", "web", "App", "HttpRequest", "HttpServer"];
+        let actix_imports = [
+            "middleware",
+            "web",
+            "App",
+            "HttpRequest",
+            "HttpServer",
+            "http",
+            "get",
+            "HttpResponse",
+            "Responder",
+        ];
```

Comment:
I'm split on how much i like this approach (importing all of the used `actix` structs/methods by name).

On one hand, it makes the code generator strings much easier to write because its just less verbeage.

On the other hand, this is pretty ugly and probably doesn't even make an impact in the end product once compiled.

So far i'll stick with this method but if the import errors get unwieldy then this is the first place that's getting the refactor hammer